project-base/src/Shopsys/ShopBundle/Component/Flysystem/VolumeDriver.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ShopBundle\Component\Flysystem;
+
+use Barryvdh\elFinderFlysystemDriver\Driver;
+use elFinder;
+use Shopsys\ShopBundle\Component\Flysystem\Plugin\GetUrl;
+
+class VolumeDriver extends Driver
+{
+    protected function configure()
+    {
+        @parent::configure();
+
+        // set thumbnails path
+        $path = $this->options['tmbPath'];
```

Comment:
I know that this elfinder really does not care about variableNames but can atleast from our side make it as understandable as possible? this is not a $path for me it is $thumbnailPath..

Maybe you could create some methods with something like `createThumbnailPath()`